src/pg/pg-store.ts

```diff
@@ -474,6 +479,10 @@ export class PgStore extends BasePgStore {
     }
   }
 
+  async getInscriptionCountPerBlock(): Promise<DbInscriptionCountPerBlock[]> {
```

Comment:
You should add some block filters here so we don't query all rows all the time

Reply:
I went a inclusive range with min/max _(0, bigint_max)_ in the query template. Curious to see what your approach is here. We could also construct a _more_ correct query, but defaults/optionals/checks were getting really ugly for this one..